[2024-09-23T18:08:44.756+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:08:43.068141+00:00 [queued]>
[2024-09-23T18:08:44.773+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:08:43.068141+00:00 [queued]>
[2024-09-23T18:08:44.775+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-09-23T18:08:44.776+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-09-23T18:08:44.777+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-09-23T18:08:44.802+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-09-23 18:08:43.068141+00:00
[2024-09-23T18:08:44.812+0000] {standard_task_runner.py:55} INFO - Started process 14760 to run task
[2024-09-23T18:08:44.824+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2024-09-23T18:08:43.068141+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpx6gtl68k']
[2024-09-23T18:08:44.829+0000] {standard_task_runner.py:83} INFO - Job 52: Subtask is_api_available
[2024-09-23T18:08:44.985+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:08:43.068141+00:00 [running]> on host 3faff16bae04
[2024-09-23T18:08:45.128+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-09-23T18:08:43.068141+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-23T18:08:43.068141+00:00
[2024-09-23T18:08:45.131+0000] {http.py:120} INFO - Poking: api/
[2024-09-23T18:08:45.159+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-09-23T18:08:45.166+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-09-23T18:08:45.574+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-09-23T18:08:45.595+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20240923T180843, start_date=20240923T180844, end_date=20240923T180845
[2024-09-23T18:08:45.681+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-09-23T18:08:45.712+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
